ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java

```diff
@@ -78,130 +82,95 @@ public void validateLink(String alias, String linkTitle) {
-//
-//		for(String title: titles) {
-//			new SyndesisRootPage().getElementContainingText()
-//		}
-//		.getElementContainingText()
-//		for (const tableTitle of tableTitlesArray) {
-//			this.expectTableTitlePresent(tableTitle, callback);
-//		}
-	}
+		String[] titles = tableTitles.split(",");
 
-	public void expectTableTitlePresent(String tableTitle, String callback) {
+		for(String title: titles) {
+			new SyndesisRootPage().getTitleByText(title).shouldBe(visible);
+		}
+	}
 
-		throw new UnsupportedOperationException();
-//    const table = this.world.app.getTitleByText(tableTitle);
-//		expect(table.isPresent(), `There must be present a table ${tableTitle}`)
-//      .to.eventually.be.true;
-//
-//		expect(table.isPresent(), `There must be enabled table ${tableTitle}`)
-//      .to.eventually.be.true.notify(callback);
+	public void expectTableTitlePresent(String tableTitle) {
+		SelenideElement table = new SyndesisRootPage().getTitleByText(tableTitle);
+		table.shouldBe(visible);
 	}
 
 	@Then("^she is presented with the \"([^\"]*)\" elements*$/")
-	public void expectElementsPresent(String elementClassNames, String callback) {
+	public void expectElementsPresent(String elementClassNames) {
+		String[] elementClassNamesArray = elementClassNames.split(",");
 
-		throw new UnsupportedOperationException();
-//    const elementClassNamesArray = elementClassNames.split(',');
-//
-//		for (const elementClassName of elementClassNamesArray) {
-//			this.expectElementPresent(elementClassName, callback);
-//		}
+		for (String elementClassName : elementClassNamesArray) {
+			SelenideElement element = new SyndesisRootPage().getElementByClassName(elementClassName);
+			element.shouldBe(visible);
+		}
 	}
 
 	@Then("^she is presented with the \"([^\"]*)\"$/")
-	public void expectElementPresent(String elementClassName, String callback) {
-
-		throw new UnsupportedOperationException();
-//    const element = this.world.app.getElementByClassName(elementClassName);
-//		expect(element.isPresent(), `There must be present a element ${elementClassName}`)
-//      .to.eventually.be.true;
-//
-//		expect(element.isPresent(), `There must be enabled element ${elementClassName}`)
-//      .to.eventually.be.true.notify(callback);
+	public void expectElementPresent(String elementClassName) {
+		SelenideElement element = new SyndesisRootPage().getElementByClassName(elementClassName);
+		element.shouldBe(visible);
 	}
 
 	@Then("^Integration \"([^\"]*)\" is present in top 5 integrations$/")
-	public void expectIntegrationPresentinTopFive(String name, String callback) {
-
-		throw new UnsupportedOperationException();
-//		log.info(`Verifying integration ${name} is present in top 5 integrations`);
-//    const page = new DashboardPage();
-//		expect(page.isIntegrationPresent(name), `Integration ${name} must be present`)
-//      .to.eventually.be.true.notify(callback);
+	public void expectIntegrationPresentinTopFive(String name) {
```

Comment:
We should probably create some separate Dashboard steps class, but it's fine now